lib/managers/docker.js

```diff
@@ -832,9 +876,21 @@ export const extractTar = async (fullImageName, dir) => {
        * 1) TAR_ENTRY_INFO is an informative error indicating that an entry is being modified.
        * 2) TAR_ENTRY_INVALID indicates that a given entry is not valid tar archive entry and will be skipped.
        */
-    } else if (!["TAR_ENTRY_INFO", "TAR_ENTRY_INVALID"].includes(err.code)) {
+    } else if (
+      DEBUG_MODE &&
+      ["TAR_ENTRY_INFO", "TAR_ENTRY_INVALID"].includes(err.code)
+    ) {
+      if (
+        err?.header?.path?.includes("{") ||
```

Comment:
We have a different bug where we attempt to untar textual metadata. This condition handles this edge case.